### PR TITLE
Meeting Minutes June

### PR DESCRIPTION
**Pull request by PatLeong**
Added June's meeting minutes, fixed .md markup, and re-ordered from
most-recent to least recent.

### CODE REVIEW

**Review by duluca — COMMENTED** [review]
All looks good

File: MeetingNotes.md [hunk 1]
```diff
@@ -1,81 +1,124 @@
...
+    3. Topic was on Wyn's Tech Radar
+    4. Pros:
+        1. No attendance limit may have helped (continue going forward)
+        2. Low rate of attrition (we expect 33%, we got high 40%'s)
+    5. Cons:
+        1. Internet went down
+2. Upcoming events
+    1. Nara Kasbergen - Empathy as a Service
+    2. Identified from WeSpeakToo.org
+    3. NPR developer
```
Comment (jenpen) [inline MeetingNotes.md:18]: I think this may be the react native developer? He is at HZDG

**Review by jenpen — COMMENTED** [review]
Looks great!

File: MeetingNotes.md (hunk 1 above)
Comment (PatLeong) [reply MeetingNotes.md:18]: nope, this is Nara, she's a developer at NPR: https://npr.codes/profile-nara-kasbergen-web-developer-6a6231b85f67

Comment (PatLeong) [reply MeetingNotes.md:18]: Oh I get it, you thought all of them were different bullets. No this is just detailing July's talk.  I didn't want to list topics we didn't approve/merge.

**Pull request merged**